### DISCUSSION TITLE
[FEATURE-REQUEST] En Automation Log no se muestran las etapas dentro de una etapa iterar (FCEach)

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#194] [3. Ideas & Feature Requests] [answered]
Buenas,

Nos percatamos que cuando se ejecuta una etapa iterar (FCEach) y hay un problema dentro de la ejecución de esas etapas, sólo se muestran los resultados de las etapas previas a la iteración. Según @rafaelwalter esto puede tener relación con la repetición de los codes de las etapas.
Ej:
0 send_message
1 iterar
1.1 create_property
1.2 create_task (error)
2 get_user

Sólo veo el result de la etapa 0 y se debería ver hasta 1.1

Saludos.

**Comment by gmoplaza** [comment]
Hola @SeIturra vamos a conversar la solicitud que haces y ver si corresponde a algo que podamos desarrollar pronto o si lo dejamos en el backlog para verlo más adelante.

**Comment by acedward** [comment, marked as answer]
@SeIturra este mes tenemos un relase que nos va a ayudar a combatir estos problema.
1. Estamos versionando los Bots paramétricos
2. Esto nos permite lanzar los Bots V2 de todos los Actuales
3. Cada bot ahora manejará los Errores de forma consistente. 

Los bots, para los errores conocidos, ya no terminarán de forma abrupta, sino que seguirán por la salida llamada "ERROR" en vez del "SUCCESS" 
Solo en casos extremos de excepciones, terminarán en forma abrupta

**Reply by acedward** [reply]
IMPORTANTE: Esto NO corrige el problema, pero lo evita en la mayoria de los casos
